GitUI/CommandsDialogs/FormViewPatch.Designer.cs

```diff
@@ -108,15 +108,13 @@ private void InitializeComponent()
             // 
             // typeDataGridViewTextBoxColumn
             // 
-            this.typeDataGridViewTextBoxColumn.DataPropertyName = "Type";
```

Comment:
💭 *Seems* like we should be fixing it here, even though the designer doesn't support `nameof`.

Reply:
Agreed, but the designer would definitely blow this away on the next rewrite. WPF is much better with this kind of binding.